lib/offline.js

```diff
@@ -55,7 +62,7 @@ module.exports = function(config) {
       }
     }
 
-    resolve(ghslug('./').catch(function() {
+    ghslug('./').catch(function() {
```

Comment:
Nit: while you're changing this line, can you move the .catch dereferencing/invocation to the next line? That'd make it more consistent with the style used elsewhere in this codebase, which is based on the style in the [Q library](https://github.com/kriskowal/q) documentation.

```js
    ghslug('./')
    .catch(function() {
```